### DISCUSSION TITLE
Student develepor pack Approved, But haven't received the benefit yet

### DISCUSSION THREAD

**Discussion by abdulwahid1o** [community/community#136131] [GitHub Education]
My application for the GitHub Student Developer Pack was approved on August 14. But, I have not yet received access to the associated benefits.Could you please assist me in resolving this issue or provide guidance on the next steps?![Screenshot_20240818-001447.png](https://github.com/user-attachments/assets/b3bf2c33-c570-416c-aa84-0282749ba777)

**Comment by neverbiasu** [comment]
U can find some relative discussion [here](https://github.com/orgs/community/discussions/55559) and the answer is [this](https://github.com/community/community/discussions/55559#discussioncomment-9795472).

**Comment by AEdgecombe** [comment]
# Answer

Check this **https://github.com/orgs/community/discussions/55559#discussioncomment-9795472**

**Comment by davevad93** [comment]
Hi @abdulwahid1o ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @abdulwahid1o was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
